build/params_calibnet.go

```diff
@@ -98,8 +98,8 @@ const UpgradePhoenixHeight = UpgradeDragonHeight + 120
 // 2024-04-03T11:00:00Z
 const UpgradeCalibrationDragonFixHeight = 1493854
 
-// ?????
-const UpgradeWaffleHeight = 999999999999999
+// 2024-07-07T12:00:00Z
```

Comment:
This date seems off - epoch seems correct

Reply:
Addressed in: https://github.com/filecoin-project/lotus/pull/12179/commits/5ced082e678959d35caad397eec0dc9b223c3d95